goodtablesio/tests/integrations/s3/models/test_s3bucket.py

```diff
@@ -90,3 +93,18 @@ def test_secret_access_key_del():
     bucket_db = database['session'].query(S3Bucket).first()
 
     assert bucket_db.secret_access_key is None
+
+
+# https://github.com/frictionlessdata/goodtables.io/issues/192
```

Comment:
It was worse - deleting an user and as consequence all his records from `source_users`. I've added with repo check.